src/Components/AnsibleTabs/AnsibleTabs.js

```diff
@@ -3,10 +3,23 @@ import './AnsibleTabs.scss';
 import { ArrowDownIcon, OutlinedFileIcon, SearchIcon } from '@patternfly/react-icons/dist/esm/icons/';
 import React, { useState } from 'react';
 import { Text, TextVariants } from '@patternfly/react-core/dist/esm/components/Text/index';
-
-import { ExpandableSection } from '@patternfly/react-core/dist/esm/components/ExpandableSection/ExpandableSection';
 import PropTypes from 'prop-types';
 import messages from '../../Messages';
+import {
+    Accordion,
+    AccordionItem,
+    AccordionToggle,
+    AccordionContent,
+    Divider,
+    Gallery,
+    Grid,
+    TextContent,
+    Tab,
+    Tabs,
+    TabTitleText,
+    Title,
+    TitleSizes
+} from '@patternfly/react-core';
```

Comment:
Can we get these imported from '/dist/ems/' makes for a smaller bundle size

Reply:
Yessir